css/templates/e-commerce/page.css

```diff
@@ -95,15 +100,14 @@ dd:after {
 
 /* TODO: Merge with existing amp-start select component */
 .commerce-select-wrapper {
-  display: inline-block;
   position: relative;
-  padding-right: 20px;
+  padding-right: 13px;
```

Comment:
Spacing to allow for down arrow in commerce-select. This shouldn't really be configurable and is used once, hence not a css var.